docs/libvirt-pod-networking.md

```diff
@@ -4,7 +4,7 @@ Author: Fabian Deutsch \<fabiand@redhat.com\>
 
 ## Introduction
```

Comment:
The main benefit is, that from the networking view, VMs and Pods can communicate with each other. They can do this by using all common known wiring constructs from Kubernetes (e.g. [Services](https://kubernetes.io/docs/concepts/services-networking/service/), [NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/), [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/), ...).

Why do you want to separate VMs and Containers? I mean there can be cases, where you don't want to have a common network, but why would the majority of people not want to have that difference abstracted away on the Service level? Especially since sophisticated overlay networks allow isolating components on the k8s level.

For instance, think about a web application, which consists of a few smaller and newer services, which can easily be put into Pods and one big monolith, which can only achieve HA, via migratable VMs? If you can't mix these on the networking level like pure podified applications, it would be pretty hard to deploy such an application and therefore.